### ISSUE TITLE
meta, name suggestion

### ISSUE THREAD

**Issue by karthik** [ropenscilabs/buffy-api#4]
Since the connection to Joss Whedon disappears without JOSS, I'd suggest just calling this robot since it would stand for ropensci bot.

**Comment by sckott** [comment]
yeah, can do that - we do need a github acct though, which we have already https://github.com/ropenscibot/ - could change that name if you want , `robot` as a gh user name is taken already though
